5_databases/week2/More_queries.sql

```diff
@@ -0,0 +1,28 @@
+SELECT * FROM user;
+
+SELECT user.name, user.email, task.title 
+FROM user JOIN user_task on user.id = user_id
+          JOIN task ON task.id= task_id
+WHERE email LIKE '%@spotify.com%';
+
+
+
+SELECT user.name, task.title, status.name 
+FROM user JOIN user_task on user.id= user_id
+          JOIN task ON task.id= task_id
+          JOIN status ON status.id= status_id
+WHERE user.name = "Donald Duck";
+
+
+
+SELECT user.name, task.title, task.created
+FROM user JOIN user_task ON user.id= user_id
+          JOIN task on task.id= task_id
+where user.name = "Maryrose Meadows"
+AND MONTH(created) =9;
```

Comment:
Make space before and after spaces so it looks like `task.id = task_id` and `= 9;`